### PR TITLE
Comments concerning the rinv bins

### PR DESCRIPTION
**Pull request by dally96**
This pull requests adds some more comments explaining why the rinv bin edges in L1Trigger/TrackFindingTracklet/interface/Settings.h were chosen as such. See here https://github.com/cms-sw/cmssw/pull/42663#discussion_r1344066992.

### CODE REVIEW

File: L1Trigger/TrackFindingTracklet/interface/Settings.h [hunk 1]
```diff
@@ -1047,6 +1047,9 @@ namespace trklet {
     double stripLength_2S_{5.0250};
 
     //Following values are used for duplicate removal
+    //The bin edges for rinv were chosen using the output tracks without duplicate removal for a large number of events
```
Comment (tomalin) [inline L1Trigger/TrackFindingTracklet/interface/Settings.h:1050]: This text is hard to understand. e.g. In "track at that index", it's unclear what the word "index2 refers to.

File: L1Trigger/TrackFindingTracklet/src/Tracklet.cc [hunk 2]
```diff
@@ -728,6 +728,15 @@ std::string Tracklet::trackfitstr() const {
 
     oss += "1|";  // valid bit
     oss += tmp.str() + "|";
+    if (settings_.combined()) {
```
Comment (tomalin) [inline L1Trigger/TrackFindingTracklet/src/Tracklet.cc:731]: Why is the C++ being changed?

**Comment by tomalin** [comment]
Changing 917 files in response to one small git review comment seems excessive! :-)

**Comment by dally96** [comment]
> Changing 917 files in response to one small git review comment seems excessive! :-)

I think I cloned the development branch and not the PR branch 😅. I'll go ahead and fix that.

**Pull request closed without merging**